pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -209,7 +208,7 @@ def _dict_to_backendinfo(cls, dct: Dict[str, Any]) -> BackendInfo:
         return BackendInfo(
             name=cls.__name__,
             device_name=name,
-            version=__extension_version__,
+            version="0.5.0",
```

Comment:
Why are we hard-coding this here?

Reply:
Oh sorry, I forgot to change that again!